### ISSUE TITLE
logger.info() not producing any output in the console

### ISSUE THREAD

**Issue by forna** [jalik/js-logger#15]
Hi, the logger.info() is not producing any output in the console.
Any idea what could be wrong? 

This is the declaration part:
```
import {
        Logger,
        INFO,
        consoleOutput
    } from '@jalik/logger'

    export const logger = new Logger({
        // Activate the logger
        active: true,
        // Set the minimal log level to log messages
        level: INFO,
        // Set the name of this logger
        name: 'main',
        // Set logging outputs
        outputs: [
            // Output logs to the console
            consoleOutput()
        ]
    })
```

Here is the usage:
```
console.log("Logger active:" + logger.active)
logger.info("I am the logger")
console.log("I am the console.log")
```

I get this result:

![image](https://user-images.githubusercontent.com/1431006/235327281-3813516b-5bfc-4cd5-9b1a-56eef6d7d28c.png)

**Comment by jalik** [comment]
Hello @forna, I've copied your code to a sandbox and it is working fine :
https://codesandbox.io/s/jalik-logger-default-example-75o5hx?file=/src/index.js

I would recommand to check first that your import is actually working, what are the values of `Logger`, `INFO` and `consoleOutput`.
If you are using a bundler (like webpack), the module resolution may not be configured properly.

**Comment by forna** [comment]
I can see you are using the 3.0.4 version, however if I install the package without specifying the version I get version 3.1.2 installed that does not work.
I have tested with 3.0.4 in my application and indeed it works.
Also, can you please check why 3.0.4 adds 150 packages (so many?) while 3.1.2 only adds 1 package?
Thanks

**v 3.1.2 (without specifying any version)**

![image](https://user-images.githubusercontent.com/1431006/235340492-2234cfec-f293-42fa-af5c-3d20435c1a98.png)

![image](https://user-images.githubusercontent.com/1431006/235340397-d4bcf342-19b8-4154-9acb-9f9790dbb3da.png)

**v 3.0.4 (by forcing the version)** 

![image](https://user-images.githubusercontent.com/1431006/235340546-b7bf0a23-3ccf-4a17-b1aa-894602a8a399.png)

![image](https://user-images.githubusercontent.com/1431006/235340457-9fb8f4b5-dbdb-46f4-9190-2f03a723cd02.png)

**Comment by jalik** [comment]
I didn't specified the latest version in the sanbox, I don't know why it selected the version 3.0.4, using the 3.1.2, the bug appears.
It could be related to these lines : https://github.com/jalik/js-logger/blob/master/src/outputs/consoleOutput.ts#L41-L42
I will see that tomorrow and publish a fix.

About the number of packages installed in the v3.0.4, it's because I decided recently to remove `@babel/runtime` from dependencies.
https://github.com/jalik/js-logger/commit/b4a571aed3d752a3fd01f5ecff7b8fb454e546fa#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L45

More info about this package : https://babeljs.io/docs/babel-runtime

**Comment by jalik** [comment]
Hello @forna , I published the 3.1.3 version which addresses this issue.
Let me know if everything works well for you.

**Comment by forna** [comment]
Yes, it works well now, thank you!
